src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java

```diff
@@ -142,11 +143,19 @@ void updateAdminPassword() {
             }
 
             // If script execution environment is set to demo, validate custom password, else if set to test, skip validation
-            if (shouldValidatePassword
-                && !ADMIN_PASSWORD.isEmpty()
-                && passwordValidator.validate(ADMIN_USERNAME, ADMIN_PASSWORD) != RequestContentValidator.ValidationError.NONE) {
-                System.out.println("Password " + ADMIN_PASSWORD + " is weak. Please re-try with a stronger password.");
-                System.exit(-1);
+            if (shouldValidatePassword && !ADMIN_PASSWORD.isEmpty()) {
+                RequestContentValidator.ValidationError response = passwordValidator.validate(ADMIN_USERNAME, ADMIN_PASSWORD);
+                if (!RequestContentValidator.ValidationError.NONE.equals(response)) {
+                    System.out.println(
+                        String.format(
+                            "Password %s failed validation: \"%s\". Please re-try with a minimum %d character password and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character that is strong. Password strength can be tested here: https://lowe.github.io/tryzxcvbn",
```

Comment:
I believe that the minimum strength can also be configured with `plugins.security.restapi.password_score_based_validation_strength`, so not sure if we want to hard code strong here? Also not sure how I feel about adding hard coded links into a log message, but maybe its fine, will defer to others here.

Reply:
More context is the feature was added in this PR: https://github.com/opensearch-project/security/issues/2569 - I pulled this setting from the PR description

Reply:
Also pretty sure the regex can be changed to not be one uppercase letter, one digit, etc. maybe we should just leave it at "meeting the minimum regex and password strength requirements"? But not sure since I think there will be a decent chunk of people who are using the minimum.

Reply:
Yea got your point. I had a misunderstanding that this was being applied to all user creation/password validation, in which case those custom settings may be present. @DarshitChanpura you are right that that would not be the case, or else they would never make it into this demo script tool. Sorry for the misleading comment.

Reply:
No worries at all, I had the same misunderstanding. Your comment wasn't misleading :)